diff --git a/dist/gitalk.js b/dist/gitalk.js
--- a/dist/gitalk.js
+++ b/dist/gitalk.js
@@ -13352,4 +13352 @@
-      var _this$options = _this.options,
-          clientID = _this$options.clientID,
-          clientSecret = _this$options.clientSecret,
-          perPage = _this$options.perPage;
+      var perPage = _this.options.perPage;
@@ -13364,5 +13361,2 @@
-            Accept: 'application/vnd.github.v3.full+json'
-          },
-          auth: {
-            username: clientID,
-            password: clientSecret
+            Accept: 'application/vnd.github.v3.full+json',
+            Authorization: _this.accessToken ? 'token ' + _this.accessToken : undefined
@@ -13708,3 +13702 @@
-          number = _options.number,
-          clientID = _options.clientID,
-          clientSecret = _options.clientSecret;
+          number = _options.number;
@@ -13716,3 +13708,2 @@
-          auth: {
-            username: clientID,
-            password: clientSecret
+          headers: {
+            Authorization: _this4.accessToken ? 'token ' + _this4.accessToken : undefined
@@ -13748,3 +13739 @@
-          labels = _options2.labels,
-          clientID = _options2.clientID,
-          clientSecret = _options2.clientSecret;
+          labels = _options2.labels;
@@ -13754,3 +13743,2 @@
-        auth: {
-          username: clientID,
-          password: clientSecret
+        headers: {
+          Authorization: this.accessToken ? 'token ' + this.accessToken : undefined
@@ -13759,0 +13748,3 @@
+          state: 'all',
+          sort: 'created',
+          direction: 'desc',
